app/views/requests/_messages.html.erb

```diff
@@ -0,0 +1,6 @@
+<% r = @request || @page || @mediated_page || @scan %>
```

Comment:
@cbeer do you want to wait for #149 so you have the helper available?  Alternatively I can fix-up this line if this is merge in before.